src/main/kotlin/uk/gov/justice/hmpps/offenderevents/config/JmsConfig.kt

```diff
@@ -89,21 +87,9 @@ class JmsConfig {
   @Bean
   @ConditionalOnProperty(name = ["sqs.provider"], havingValue = "localstack")
   @Suppress("SpringJavaInjectionPointsAutowiringInspection")
-  open fun queueUrl(
+  fun queueUrl(
     @Autowired awsSqsClient: AmazonSQS,
     @Value("\${sqs.queue.name}") queueName: String,
     @Value("\${sqs.dlq.name}") dlqName: String
-  ): String {
-    val result = awsSqsClient.createQueue(CreateQueueRequest(dlqName))
```

Comment:
don't use embedded localstack so don't need to create queues in the kotlin any more